public/include/function.php

```diff
@@ -1314,20 +1314,19 @@ function selectHeure($min, $max, $blank=false, $selectedValue=null)
     }
 }
 
-function myselectHeure($min, $max, $blank=false, $selectedValue=null)
+function myselectHeure($min, $max, $blank=false)
```

Comment:
I created a new function to manage select options when the user chooses hours for their table.